wgs/community-membership.md

```diff
@@ -96,8 +96,8 @@ The following apply to the part of codebase for which one would be a reviewer in
 an [OWNERS] file (for repos using the bot).
 
 - member for at least 3 months
-- Primary reviewer for at least 5 PRs to the codebase
-- Reviewed or merged at least 20 substantial PRs to the codebase
+- Primary reviewer for at least 3 PRs to the codebase
+- Reviewed or merged at least 5 substantial PRs to the codebase
```

Comment:
I would clarify what substantial means:

Perhaps add something like "with the definition of substantial subject to the lead's discretion (e.g. refactors, enhancements rather than grammar correction or one-line pulls)" similar to what we are doing at Argo project.

Reply:
+1. As in the [Argo doc](https://github.com/argoproj/argoproj/blob/master/community/membership.md), we can also combine these 2 points to one `Reviewer for or author of at least 5 substantial PRs to the codebase,`

Reply:
+1 to Johnu's wording.

Reply:
ok, I changed to "Reviewer for or author of at least 5 substantial PRs to the codebase" in 2nd comment